moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -58,6 +58,63 @@ static double normalizeAngle(double angle)
   return v;
 }
 
+// Normalizes an angles to an absolute angle.
```

Comment:
```suggestion
// Normalizes an angle to the interval [-pi, +pi] and then take the absolute value
```

I know this explanation is not how the code does it, but I think it better explains what is happening. Or am I wrong in what the code does?